apps/communications/contacts/locales/matcher/matcher.en-US.properties

```diff
@@ -9,17 +9,17 @@ curtainCancel                 = Cancel
 merge                         = Merge
 close                         = Ignore
 
-mergeDuplicateTitle           = Merge duplicates
-suggestedDuplicateContacts    = Suggested duplicate contacts for {{name}}
-mergeActionButtonLabel        = {[ plural(n) ]}
-mergeActionButtonLabel[zero]  = No contacts selected
-mergeActionButtonLabel[one]   = Merge with {{n}} contact
-mergeActionButtonLabel[two]   = Merge with {{n}} contacts
-mergeActionButtonLabel[few]   = Merge with {{n}} contacts
-mergeActionButtonLabel[many]  = Merge with {{n}} contacts
-mergeActionButtonLabel[other] = Merge with {{n}} contacts
+mergeDuplicateTitle                  = Merge duplicates
+suggestedDuplicatesMessage.innerHTML = Suggested duplicate contacts for <bdi>{{name}}</bdi>
```

Comment:
The problem here is that contact name/intl. number is used inside l10n string. At the moment I can think of the following solutions:
* Use ```<bdi>``` in l10n file. Recently l10n team implemented DOM Overlays [1], so that we can get rid of ```.innerHTML``` thing in ```.properties``` files. But it's not likely that it will be uplifted to v2.2 and even in master it still depends on [2], so Stas recommended to stick with ```.innerHTML``` for both master and v2.2 for now.

* Use Unicode v6.3 isolation marks (FSI/PDI) - Gecko doesn't support it yet (see [3]);

* Redesign this top message so that contact name is rendered in a separate element, like subheader or smth similar - too risky and not necessary. I named it as _redesign_ because of the fact that we can't just extract contact name variable from l10n string as we don't know its correct position in all locales, eg. compare [4] - English, [5] - Polish and [6] - Lithuanian;

So, IMO the first option is the best, but what do you think?

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=994357
[2] https://bugzilla.mozilla.org/show_bug.cgi?id=1027117
[3] https://bugzilla.mozilla.org/show_bug.cgi?id=922963
[4] http://mxr.mozilla.org/l10n-gaia/source/en-US/apps/communications/contacts/matcher/matcher.properties#13
[5] http://mxr.mozilla.org/l10n-gaia/source/pl/apps/communications/contacts/matcher/matcher.properties#10
[6] http://mxr.mozilla.org/l10n-gaia/source/lt/apps/communications/contacts/matcher/matcher.properties#13